Makefile

```diff
@@ -69,22 +69,29 @@ infrastructure-core: bootstrap ## Deploy core infrastructure
 infrastructure-transform: bootstrap transform-artifacts ## Deploy transform infrastructure
 	$(call terragrunt,apply,infrastructure/transform)
 
+infrastructure-and-test-transform: infrastructure test-transform
```

Comment:
why would you want to deploy serve but only test transform?

Reply:
I think this is actually unused now - I added this during my several unsuccessful iterations of trying to pick the right Make target to run this script 😫